src/evict/evict_page.c

```diff
@@ -624,56 +624,33 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
      * memory.
      */
     cache = conn->cache;
-    lookaside_retry = false;
-    lookaside_retryp = NULL;
 
     if (closing)
         LF_SET(WT_REC_VISIBILITY_ERR);
-    else if (WT_PAGE_IS_INTERNAL(page) || WT_IS_LAS(S2BT(session)))
+
+    if (WT_PAGE_IS_INTERNAL(page) || WT_IS_LAS(S2BT(session)))
```

Comment:
This change doesn't make sense to me, can you explain what's going on here?

Reply:
This is to force we do lookaside also when closing.

Reply:
I agree this doesn't make sense. I'll revert that. The whole tree should be clean when closing. However, I occasionally hit 

```
WT_ASSERT(
      session, !__wt_page_is_modified(page) || LF_ISSET(WT_REC_LOOKASIDE | WT_REC_IN_MEMORY));
```

in evict_page.c